### PR TITLE
[SYCL] Test that the UUID is read correctly from Level Zero

### PR DESCRIPTION
**Pull request by smaslov-intel**
Tests this change : https://github.com/intel/llvm/pull/8088/commits/28588ae3e91816dbeaa4fb46a3b3b86e98a8ce76
Signed-off-by: Sergey V Maslov <sergey.v.maslov@intel.com>

### CODE REVIEW

**Review by againull — APPROVED** [review]

File: SYCL/Plugin/level_zero_uuid.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// REQUIRES: gpu, level_zero, level_zero_dev_kit
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+
+// Test that the UUID is read correctly from Level Zero.
+
```
Comment (smaslov-intel) [inline SYCL/Plugin/level_zero_uuid.cpp:7]:
```suggestion

// CHECK: PASSED
```

**Comment by smaslov-intel** [comment]
@intel/llvm-gatekeepers : this can be merged I believe

**Comment by bader** [comment]
> @intel/llvm-gatekeepers : this can be merged I believe

@smaslov-intel, there are no test results from Jenkins CI. Please, contact tools team to fix it.

**Comment by steffenlarsen** [comment]
> > @intel/llvm-gatekeepers : this can be merged I believe
> 
> @smaslov-intel, there are no test results from Jenkins CI. Please, contact tools team to fix it.

Apologies. I missed this. The CI problem mentioned was fixed a couple of days ago and this commit may have been missed by the people manually restarting affected PRs.

**Pull request merged**